taccsite_cms/settings.py

```diff
@@ -320,7 +320,8 @@ def gettext(s): return s
 def get_subdirs_as_module_names(path):
     module_names = []
     for entry in os.scandir(path):
-        if entry.is_dir():
+        is_node_modules = entry.path.find('node_modules') > -1
+        if entry.is_dir() and not is_node_modules:
```

Comment:
**Reminder.** This prevents build error after Django tries to read `taccsite_custom/node_modules` as an app.

Reply:
This change is absent in upcoming PR.